File: ts-sdk/src/sdk/models/operations/deletefilesidmetadataenterprisesecurityclassification6vmvochwuwo.ts
import { SpeakeasyMetadata, SpeakeasyBase, RetryConfig } from "../../../internal/utils";
import * as shared from "../shared";



export class DeleteFilesIdMetadataEnterpriseSecurityClassification6VmVochwUWoPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=file_id" })
  fileId: string;
}


export class DeleteFilesIdMetadataEnterpriseSecurityClassification6VmVochwUWoRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  retries?: RetryConfig;

  @SpeakeasyMetadata()
  pathParams: DeleteFilesIdMetadataEnterpriseSecurityClassification6VmVochwUWoPathParams;
}


export class DeleteFilesIdMetadataEnterpriseSecurityClassification6VmVochwUWoResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  clientError?: shared.ClientError;

  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}

File: sdks/go/pkg/models/operations/deletefilesidmetadataenterprisesecurityclassification6vmvochwuwo.go
package operations

import (
	"openapi/pkg/models/shared"
	"openapi/pkg/utils"
)

type DeleteFilesIDMetadataEnterpriseSecurityClassification6VMVochwUWoPathParams struct {
	FileID string `pathParam:"style=simple,explode=false,name=file_id"`
}

type DeleteFilesIDMetadataEnterpriseSecurityClassification6VMVochwUWoRequest struct {
	Retries    *utils.RetryConfig
	PathParams DeleteFilesIDMetadataEnterpriseSecurityClassification6VMVochwUWoPathParams
}

type DeleteFilesIDMetadataEnterpriseSecurityClassification6VMVochwUWoResponse struct {
	ClientError *shared.ClientError
	ContentType string
	StatusCode  int64
}
